config/scenario_config_ENGAGE_4p5.csv

```diff
@@ -1,6 +1,6 @@
 title;start;slurmConfig;climate;cm_magicc_calibrateTemperature2000;cm_damage_KWSE;cm_magicc_config;cm_magicc_temperatureImpulseResponse;cm_damage_DiceLike_specification;cm_damages_BurkeLike_persistenceTime;cm_damages_BurkeLike_specification;cm_damages_SccHorizon;cm_VRE_supply_assumptions;c_CES_calibration_new_structure;buildings;transport;industry;cm_DiscRateScen;c_shBioTrans;cm_EDGEtr_scen;cm_reducCostB;cm_CES_calibration_default_prices;c_ccsinjecratescen;.CDR;cm_bioenergy_SustTax;cm_rcp_scen;cm_iterative_target_adj;subsidizeLearning;c_budgetCO2from2020;carbonprice;carbonpriceRegi;cm_netZeroScen;cm_co2_tax_2020;cm_peakBudgYr;cm_taxCO2inc_after_peakBudgYr;cm_CO2priceRegConvEndYr;cm_emiscen;c_regi_earlyreti_rate;c_tech_earlyreti_rate;cm_fetaxscen;cm_co2_tax_growth;cm_maxProdBiolc;c_ccscapratescen;techpol;c_techAssumptScen;cm_nucscen;cm_so2tax_scen;cm_multigasscen;cm_LU_emi_scen;cm_tradecostBio;cm_1stgen_phaseout;c_SSP_forcing_adjust;cm_APscen;water;cm_startyear;path_gdx;path_gdx_ref;path_gdx_bau;description
 SSP2-Base_bIT;0;5;off;uncalibrated;0;RCP26_50;off;HowardNonCatastrophic;15;0;100;0;0;simple;edge_esm;subsectors;0;1;Mix1;none;0.01;1;off;1.5;none;0;off;0;none;none;;-1;2100;3;2050;1;;;3;1.05;off;1;none;1;2;1;2;SSP2;1;0;forcing_SSP2;SSP2;heat;2005;;;;SSP2-Base_bIT: This baseline scenario follows the Shared Socioeconomic Pathways 2 called Middle of the Road. Industry sectors are modeled explicitly with individual CES nests for cement, chemicals, steel, and other production. The transport model EDGE-T with detailed modes/vehicles representation is used. A simple buildings model represents demand in terms of energy carriers. No Damages from climate change are considered.
-GP_CurPol_T45;1;5;off;uncalibrated;0;RCP26_50;off;HowardNonCatastrophic;30;0;100;0;0;simple;edge_esm;subsectors;0;1;Mix3;none;0.01;1;off;1.5;rcp45;3;off;0;Npi;none;;1;2100;3;2050;9;;;3;1.05;100;1;NPi2018;1;2;1;3;SSP2;1;0;forcing_SSP2;SSP2;heat;2005;;;;GP_CurPol_T45: The Current Policies scenarios describe energy, climate and economic projections for the period until 2030.
+GP_CurPol_T45;1;5;off;uncalibrated;0;RCP26_50;off;HowardNonCatastrophic;30;0;100;0;0;simple;edge_esm;subsectors;0;1;Mix3;none;0.01;1;off;1.5;rcp45;3;off;0;NPi;none;;1;2100;3;2050;9;;;3;1.05;100;1;NPi2018;1;2;1;3;SSP2;1;0;forcing_SSP2;SSP2;heat;2005;;;;GP_CurPol_T45: The Current Policies scenarios describe energy, climate and economic projections for the period until 2030.
```

Comment:
sometimes, my excel did not want to take NPi and forced me to Npi, I also fixed it quite often thanks for identifying more